examples/src/utils.cpp

```diff
@@ -114,7 +114,12 @@ la::avdecc::protocol::ProtocolInterface::Type chooseProtocolInterfaceType(la::av
 		int index = -1;
 		while (index == -1)
 		{
-			int c = getch() - '0';
+#if defined(USE_CURSES)
```

Comment:
Thanks you very much for this. Screen being cleared has been an issue I've lived with for many years without having found the time to search for the reason (nor a fix).
I'll create a preprocessor macro though, instead of having to use if preprocessor conditions all over the place.

Reply:
I'll fix it in my PR. How would you like it to look?

Reply:
Just a simple
`
#if defined(USE_CURSES)
#define GETCH() wgetch(s_Window)
#else 
#define GETCH() getch()
#endif `

I think it should do the trick

Keep this fix as a separate commit as I won't be able to integrate the other fixes right now, only this one is a perfect fit currently.

Reply:
@christophe-calmejane I've reopened new PR #71 with a single commit. Don't you mind merging it with GitHub, rather than by hand? This is crucial thing for me.